external-import/harfanglab-incidents/README.md

```diff
@@ -0,0 +1,137 @@
+- pycti >= 6.4.X
+
+## Configuration variables
+
+There are a number of configuration options, which are set either in `docker-compose.yml` (for Docker) or
+in `config.yml` (for manual deployment).
+
+### OpenCTI environment variables
+
+Below are the parameters you'll need to set for OpenCTI:
+
+| Parameter     | config.yml `opencti` | Docker environment variable | Mandatory | Description                                          |
+|---------------|----------------------|-----------------------------|-----------|------------------------------------------------------|
+| OpenCTI URL   | url                  | `OPENCTI_URL`               | Yes       | The URL of the OpenCTI platform.                     |
+| OpenCTI Token | token                | `OPENCTI_TOKEN`             | Yes       | The default admin token set in the OpenCTI platform. |
+
+### Base connector environment variables
+
+Below are the parameters you'll need to set for running the connector properly:
+
+| Parameter       | config.yml `connector` | Docker environment variable | Default         | Mandatory | Description                                                                              |
+|-----------------|------------------------|-----------------------------|-----------------|-----------|------------------------------------------------------------------------------------------|
+| Connector ID    | id                     | `CONNECTOR_ID`              | /               | Yes       | A unique `UUIDv4` identifier for this connector instance.                                |
+| Connector Type  | type                   | `CONNECTOR_TYPE`            | EXTERNAL_IMPORT | Yes       | Should always be set to `EXTERNAL_IMPORT` for this connector.                            |
+| Connector Name  | name                   | `CONNECTOR_NAME`            |                 | Yes       | Name of the connector.                                                                   |
+| Connector Scope | scope                  | `CONNECTOR_SCOPE`           |                 | Yes       | The scope or type of data the connector is importing, either a MIME type or Stix Object. |
+| Log Level       | log_level              | `CONNECTOR_LOG_LEVEL`       | info            | Yes       | Determines the verbosity of the logs. Options are `debug`, `info`, `warn`, or `error`.   |
+
+### Connector extra parameters environment variables
+
+Below are the parameters you'll need to set for the connector:
+
+| Parameter    | config.yml `harfanglab_incidents` | Docker environment variable | Default | Mandatory | Description |
+|--------------|-----------------------------------|-----------------------------|---------|-----------|-------------|
+| API base URL | api_base_url                      |                             |         | Yes       |             |
+| API key      | api_key                           |                             |         | Yes       |             |
```

Comment:
I think this should be updated

Reply:
I updated most of the README, is it better?